tests/models/test_dagbag.py

```diff
@@ -194,13 +194,14 @@ def test_get_dag_fileloc(self):
         Test that fileloc is correctly set when we load example DAGs,
         specifically SubDAGs and packaged DAGs.
         """
-        dagbag = models.DagBag(include_examples=True)
+        dagbag = models.DagBag(dag_folder=self.empty_dir, include_examples=True)
```

Comment:
Looks like a great change to me.

One slight nit: I think that you may be able to create this `dagbag` object in the `setUp` method, which would be slightly more DRY.

Reply:
Thanks for taking a look.

The convention in this file seems to be to create the dagbag in each test method.  

I think this is reasonable because in different tests methods, DagBag will be instantiated in different ways; some include examples (or not), some say safe mode (or not), some reference a dag_folder (or not).  And the cost of repeating it is very low -- around 1 line.  And the benefit of repeating it is you can see, in each test method, exactly what it is doing.  

What do you think?

Reply:
Agreed. :+1: